web/Controllers/AdminController.cs

```diff
@@ -26,4 +29,40 @@ public IActionResult Index()
 
         return View(viewModel);
     }
+
+    [AllowAnonymous]
+    [HttpPost]
+    public IActionResult RequestAdminAccess()
+    {
+        var isAuthenticated = User.Identity?.IsAuthenticated ?? false;
+
+        if (!isAuthenticated)
+        {
+            _logger.LogWarning("Not authenticated user");
+            TempData["FlashType"] = "danger";
+

```

Comment:
Unnecessary line break - remove to be consistent with the same pattern  you've used below :)